spec/middleware/authentication_spec.rb

```diff
@@ -16,6 +16,15 @@
 
         expect(status).to eq(200)
       end
+
+      it 'allows the request if given a correct password header and a Passwords header' do
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.